modules/ob-creating-a-buildah-build.adoc

```diff
@@ -12,11 +12,14 @@ You can create a `buildah` build and push the created image to the target regist
 
 * You have installed the {builds-operator} on the {ocp-product-title} cluster.
 * You have created a `ShipwrightBuild` resource.
+* You have installed the `oc` CLI.
+* You have installed the `shp` CLI, if needed.
 
 .Procedure
 
 . Create a `Build` resource and apply it to the {ocp-product-title} cluster:
```

Comment:
The lead in sentence should state to choose one of the following CLI options. Might be obvious, but best to state it.